syntactic/src/main/scala/syntactic/whitelist/WhitelistChecker.scala

```diff
@@ -16,7 +16,7 @@ class WhitelistChecker private(allowedFeatures: List[Feature]) extends Checker {
 
   override def checkTree(tree: Tree): Option[Violation] = {
     if (GroupedFeature.allows(tree)) None
-    else Some(Violation(tree))
+    else Some(Violation(tree, s"not in the allowed features: ${GroupedFeature.toString}"))
```

Comment:
I think that it would be better to exclude `AlwaysAllowed` from the list of allowed features when displayed to the user.

Reply:
Makes sense. But if a whitelist checker is initialized with an empty list, the features displayed will be empty, unless features are required to be non-empty like blacklist checkers. Is there a scenario where users want to use the checker which allows only `AlwaysAllowed`?

Reply:
No, `AlwaysAllowed` on its own does not allow to write anything (at least nothing meaningful). So I think that we can require that there is at least 1 feature. Also we may sort the features'names by alphabetical order to make it easier for the user, but that is a detail

Reply:
Done. AlwaysAllowed is not displayed and feature names are sorted.